packages/wallets/src/features/cashier/flows/WalletWithdrawal/WalletWithdrawal.tsx

```diff
@@ -1,15 +1,23 @@
 import React, { useEffect, useState } from 'react';
-import { useActiveWalletAccount, useAuthorize } from '@deriv/api-v2';
+import { useActiveWalletAccount, useAuthorize, useBalance } from '@deriv/api-v2';
 import { Loader } from '../../../../components';
 import { WithdrawalCryptoModule, WithdrawalFiatModule, WithdrawalVerificationModule } from '../../modules';
 import { WithdrawalNoBalance } from '../../screens';
 
 const WalletWithdrawal = () => {
     const { switchAccount } = useAuthorize();
     const { data: activeWallet } = useActiveWalletAccount();
+    const { data: balanceData, isLoading, isRefetching, refetch } = useBalance();
```

Comment:
Maybe worth adding a TODO to refactor this later as tech debt, and use the balance from subscription in the useActiveWalletAccount hook.

Reply:
Will add it in this PR: https://github.com/binary-com/deriv-app/pull/15350